utils/cli/multisend.go

```diff
@@ -0,0 +1,225 @@
+					return fmt.Errorf("transaction failed")
+				}
+				return nil
+			}
+
+			expectedSequenceOrigin, err := getSequence(clientCtxOrigin.FromAddress.String())
+			if err != nil {
+				return err
+			}
+
+			expectedSequenceHW := expectedSequenceOrigin
+			if useHotWallet {
+				expectedSequenceHW, err = getSequence(clientCtxHotWallet.FromAddress.String())
+				if err != nil {
+					return err
+				}
+			}
+
+			output := []banktypes.Output{}
+			totalAmount := sdk.Coins{}
+			records = records[startIndex:]
+			for i, record := range records {
+				coins, err := sdk.ParseCoinsNormalized(record[1])
+				if err != nil {
+					fmt.Printf("failed sending records from %d to %d\n", startIndex, i)
+					fmt.Printf("please run again with: multi-send [file.csv] %d\n", startIndex)
+					return err
+				}
+
+				if coins.IsZero() {
+					fmt.Printf("failed sending records from %d to %d\n", startIndex, i)
+					fmt.Printf("please run again with: multi-send [file.csv] %d\n", startIndex)
+					return fmt.Errorf("must send positive amount")
+				}
+				toAddr, err := sdk.AccAddressFromBech32(record[0])
+				if err != nil {
+					fmt.Printf("failed sending records from %d to %d\n", startIndex, i)
+					fmt.Printf("please run again with: multi-send [file.csv] %d\n", startIndex)
+					return err
+				}
+				output = append(output, banktypes.NewOutput(toAddr, coins))
+				totalAmount = totalAmount.Add(coins...)
+
+				if (i+1)%MAX_ADDRESSES == 0 || (i+1) == len(records) {
+					if useHotWallet {
+						msg := banktypes.NewMsgSend(clientCtxOrigin.FromAddress, clientCtxHotWallet.FromAddress, totalAmount.Add(sdk.NewCoin(commontypes.TokenDenom, sdk.NewInt(5))))
+						err := tx.GenerateOrBroadcastTxCLI(clientCtxOrigin, cmd.Flags(), msg)
+						if err != nil {
+							fmt.Printf("failed sending records from %d to %d\n", startIndex, i)
+							fmt.Printf("please run again with: multi-send [file.csv] %d\n", startIndex)
+							return err
+						}
+						expectedSequenceOrigin++
+						err = waitSequenceChange(clientCtxOrigin.FromAddress.String(), expectedSequenceOrigin)
+						if err != nil {
+							fmt.Printf("failed sending records from %d to %d\n", startIndex, i)
+							fmt.Printf("please run again with: multi-send [file.csv] %d\n", startIndex)
+							return err
+						}
+					}
+
+					err = waitSequenceChange(clientCtxHotWallet.FromAddress.String(), expectedSequenceHW)
+					if err != nil {
+						fmt.Printf("failed sending records from %d to %d\n", startIndex, i)
+						fmt.Printf("please run again with: multi-send [file.csv] %d\n", startIndex)
+						return err
+					}
+					msg := banktypes.NewMsgMultiSend([]banktypes.Input{banktypes.NewInput(clientCtxHotWallet.FromAddress, totalAmount)}, output)
+					err := tx.GenerateOrBroadcastTxCLI(clientCtxHotWallet, cmd.Flags(), msg)
+					if err != nil {
+						fmt.Printf("failed sending records from %d to %d\n", startIndex, i)
+						fmt.Printf("please run again with: multi-send [file.csv] %d\n", startIndex)
```

Comment:
let's add a warning to check the funds in the hot wallet, because if they are gone we can't run with the same index